basically running and manually verifying result:

grep -r '"docs/' . -l |xargs sed -i -e 's,"docs/,"https://www.phorum.org/phorum5/docs/,g'

--- phorum-5.2.23/common.php~	2020-11-24 12:23:47.000000000 +0200
+++ phorum-5.2.23/common.php	2020-11-24 12:25:57.349551331 +0200
@@ -103,7 +103,7 @@
         if (!file_exists("/etc/webapps/phorum/config.php")) { ?>
             Phorum has been installed on this server, but the configuration<br />
             for the database connection has not yet been made. Please read<br />
-            <a href="docs/install.txt">docs/install.txt</a> for installation
+            <a href="https://www.phorum.org/phorum5/docs/install.txt">https://www.phorum.org/phorum5/docs/install.txt</a> for installation
             instructions. <?php
         } else {
             $fp = fopen("/etc/webapps/phorum/config.php", "r");
--- phorum-5.2.23/include/admin/install.php~	2020-11-24 12:23:47.000000000 +0200
+++ phorum-5.2.23/include/admin/install.php	2020-11-24 12:25:13.710631576 +0200
@@ -520,7 +520,7 @@
 
         $cont_url = phorum_admin_build_url('');
         phorum_db_update_settings( array("installed"=>1) );
-        echo "The setup is complete.  You can now go to <a href=\"$cont_url\">the admin</a> and start making Phorum all your own.<br /><br /><strong>Here are some things you will want to look at:</strong><br /><br /><a href=\"$_SERVER[PHP_SELF]?module=settings\">The General Settings page</a><br /><br /><a href=\"$_SERVER[PHP_SELF]?module=mods\">Pre-installed modules</a><br /><br /><a href=\"docs/faq.txt\">The FAQ</a><br /><br /><a href=\"docs/performance.txt\">How to get peak performance from Phorum</a><br /><br /><strong>For developers:</strong><br /><br /><a href=\"docs/creating_mods.txt\">Module Creation</a><br /><br /><a href=\"docs/permissions.txt\">How Phorum permisssions work</a><br /><br /><a href=\"docs/CODING-STANDARDS\">The Phorum Team&apos;s codings standards</a>";
+        echo "The setup is complete.  You can now go to <a href=\"$cont_url\">the admin</a> and start making Phorum all your own.<br /><br /><strong>Here are some things you will want to look at:</strong><br /><br /><a href=\"$_SERVER[PHP_SELF]?module=settings\">The General Settings page</a><br /><br /><a href=\"$_SERVER[PHP_SELF]?module=mods\">Pre-installed modules</a><br /><br /><a href=\"https://www.phorum.org/phorum5/docs/faq.txt\">The FAQ</a><br /><br /><a href=\"https://www.phorum.org/phorum5/docs/performance.txt\">How to get peak performance from Phorum</a><br /><br /><strong>For developers:</strong><br /><br /><a href=\"https://www.phorum.org/phorum5/docs/creating_mods.txt\">Module Creation</a><br /><br /><a href=\"https://www.phorum.org/phorum5/docs/permissions.txt\">How Phorum permisssions work</a><br /><br /><a href=\"https://www.phorum.org/phorum5/docs/CODING-STANDARDS\">The Phorum Team&apos;s codings standards</a>";
 
         break;
 
